docs/data-cookbooks/cnm-workflow.md

```diff
@@ -42,7 +42,7 @@ You should be able to quickly use the "Create Data Stream" button on the [Kinesi
 
 ![Screenshot of AWS console page for creating a Kinesis stream](assets/cnm_create_kinesis_stream.jpg)
 
-Please bear in mind that your `{{prefix}}-lambda-processing` IAM role will need permissions to write to the response stream for this workflow to succeed if you create the Kinesis stream with a dashboard user.   If you are using the example deployment (or a deployment based on it), the IAM permissions should be set properly.
+Please bear in mind that your `{{prefix}}-lambda-processing` IAM role will need permissions to write to the response stream for this workflow to succeed if you create the Kinesis stream with a dashboard user.   If you are using the `cumulus` top-level module for your deployment this should be set properly.
```

Comment:
This role still exists/is relevant?

Reply:
Yes, I think so: https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/iam.tf